src/components/ChoiceList/tests/ChoiceList.test.tsx

```diff
@@ -16,22 +23,40 @@ describe('<ChoiceList />', () => {
     ];
   });
 
-  it('renders a fieldset', () => {
-    const element = mount(<ChoiceList selected={[]} choices={choices} />);
+  it('should verify renders a fieldset', () => {
```

Comment:
Add this 'it' statement into `describe('When default props are provided', () => {`